lib/miio/Status.js

```diff
@@ -4,16 +4,17 @@
  */
 const Status = function() {
     /** List of allowed values as per Roborock.STATUS_MAP */
-    this.state = "UNKNOWN";
+    this.state = 0;
     this.error_code = null;
     this.battery = null;
     this.clean_time = null;   // Seconds?
     this.clean_area = null;   // mm²
     this.error_code = null;   // TODO
     this.map_present = false; // TODO
-    this.in_cleaning = false; // TODO
+    this.in_cleaning = 0; // TODO
```

Comment:
Why int?

Reply:
That was only a fix due to type checking.
There is Code which uses it as number.

Reply:
It is used everywhere as a number.
Maybe it should be refactored later, it's not the topic of this pull request.